tests/e2e/linalg_ext_ops/CMakeLists.txt

```diff
@@ -30,6 +30,25 @@ iree_check_single_backend_test_suite(
     "requires-gpu-nvidia"
 )
 
+iree_check_single_backend_test_suite(
+  NAME
+    check_cuda-topk-split-reduction
+  SRCS
+    "top-k-split-reduction.mlir"
+  TARGET_BACKEND
+    "cuda"
+  DRIVER
+    "cuda"
+  COMPILER_FLAGS
+    "--iree-flow-topk-split--reduction=3"
```

Comment:
3 doesn't divice the input dimension in the test below. Are you sure split K is actually triggered?

Reply:
changed to 2. Verified to run on the `topk_1d_dim0_max_optional` in  `top-k.mlir`.